test/parallel/test-cluster-inspect-port-manual.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+/**
+ * This test suite checks debug port numbers
+ * when node started without --inspect argument
+ * and debug initiated with cluster.settings.execArgv or similar
+ */
+
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+
+if (cluster.isMaster) {
+  let message = 'If master is not debugging, ' +
+                'forked worker should not have --inspect argument';
+
+  cluster.fork({message}).on('exit', common.mustCall(checkExitCode));
+
+  message = 'When debugging port is set at cluster.settings, ' +
+            'forked worker should have debugPort without offset';
+  cluster.settings.execArgv = ['--inspect=' + common.PORT];
```

Comment:
> Should we just move tests, which do, to /sequential/?

Tests that use `common.PORT`? Ideally, yes.

Tests that use port `0`? No, those should stay in `/parallel/`.